packages/components/src/internal/components/chart/Chart.tsx

```diff
@@ -115,6 +118,7 @@ export const SVGChart: FC<Props> = memo(({ api, chart, container, filters }) =>
             if (filters) {
                 queryConfig_.filterArray = [...queryConfig_.filterArray, ...filters];
             }
+            queryConfig_.parameters = model?.queryParameters;
```

Comment:
Perhaps just hand through the `model.queryParameters` and then there is no need to pass through the entire `QueryModel`.

Reply:
yes, I'll make that change. I had originally thought that I'd pass the model and then use that to get the filters and queryParameters.